parmed/amber/_amberparm.py

```diff
@@ -316,8 +319,8 @@ def from_structure(cls, struct):
         inst.LJ_radius = [0 for i in range(ntyp)]
         inst.LJ_depth = [0 for i in range(ntyp)]
         for atom in inst.atoms:
-            inst.LJ_radius[atom.nb_idx-1] = atom.atom_type.rmin
-            inst.LJ_depth[atom.nb_idx-1] = atom.atom_type.epsilon
+            inst.LJ_radius[atom.nb_idx - 1] = atom.atom_type.rmin
+            inst.LJ_depth[atom.nb_idx - 1] = atom.atom_type.epsilon
```

Comment:
yeah, I think both of them are fine.

https://www.python.org/dev/peps/pep-0008/
```python
Yes:

ham[1:9], ham[1:9:3], ham[:9:3], ham[1::3], ham[1:9:]
ham[lower:upper], ham[lower:upper:], ham[lower::step]
ham[lower+offset : upper+offset]
ham[: upper_fn(x) : step_fn(x)], ham[:: step_fn(x)]
ham[lower + offset : upper + offset]
```